app/api/files/jsRoutes.js

```diff
@@ -46,13 +46,23 @@ export default app => {
     '/api/public',
     multer().any(),
     captchaAuthorization(),
-    (req, _res, next) => {
-      req.body = JSON.parse(req.body.entity);
-      return next();
-    },
-    validation.validateRequest(saveSchema),
+    validation.validateRequest({
+      entity: {
```

Comment:
Every other route that uses the non-Joi validation has a structure of the type:
```
{
  porperties: {
    *key* (ie: entity): {
      poperites: {
        *key* (ie: title) : { type: 'string' }
      }
  }
}
```
Is this form a shorthand and the extra keys are no longer required?  If so, why are we using them on every other route?

Reply:
this validation probably fails, the idea is to actually include in the validator which part of the req you are validating, body, query, params or any other in this case probably 
```
{
  properties: {
    body: {
      properties: {
        *key* (ie: title) : { type: 'string' }
      }
  }
}
```

also we already have an entity schema json defined that i think should be used if this is expected to be a full entity ?

Reply:
The validation wasn't failing probably because without the "properties" key it just ignores everything and doesn't matter what you send.